### PR TITLE
Added missing chart value for condition configuration to helm chart

### LINKED ISSUE #1444: Condition configuration missing in helm chart
(Additional) Shoot Readiness conditions can be configured or overwritten via dashboard configuration. However this is not implemented in the helm chart.

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #1444

**Special notes for your reviewer**:

**Release note**:

```bugfix operator
Added missing chart value for condition configuration to helm chart 
```

### CODE REVIEW

File: charts/__tests__/gardener-dashboard/runtime/dashboard/configmap.spec.js [hunk 1]
```diff
@@ -585,5 +585,32 @@ describe('gardener-dashboard', function () {
...
+          global: {
+            dashboard: {
+              frontendConfig: {
+                knownConditions: {
+                  ExampleConditionReady: {
+                  name: 'Example',
+                  shortName: 'E',
+                  description: 'Example Description',
+                  showAdmin: false,
+                  sortOrder: '99'
```
Comment (holgerkoser) [inline charts/__tests__/gardener-dashboard/runtime/dashboard/configmap.spec.js:601]:
```suggestion
                    name: 'Example',
                    shortName: 'E',
                    description: 'Example Description',
                    showAdmin: false,
                    sortOrder: '99'
```

File: charts/__tests__/gardener-dashboard/runtime/dashboard/__snapshots__/deployment.spec.js.snap [hunk 2]
```diff
@@ -146,6 +146,10 @@ Object {
                 "name": "GARDENER_CONFIG",
                 "value": "/etc/gardener-dashboard/config.yaml",
               },
+              Object {
```
Comment (holgerkoser) [inline charts/__tests__/gardener-dashboard/runtime/dashboard/__snapshots__/deployment.spec.js.snap:149]: I assume the chart tests have failed or why this is part of your PR?

File: charts/__tests__/gardener-dashboard/runtime/dashboard/deployment.spec.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ describe('gardener-dashboard', function () {
         expect(container.volumeMounts).toHaveLength(4)
         const [, , , kubeconfigVolumeMount] = container.volumeMounts
         expect(kubeconfigVolumeMount).toMatchSnapshot()
-        expect(container.env).toHaveLength(5)
+        expect(container.env).toHaveLength(6)
```
Comment (holgerkoser) [inline charts/__tests__/gardener-dashboard/runtime/dashboard/deployment.spec.js:121]: Same as above?

File: charts/gardener-dashboard/values.yaml [hunk 4]
```diff
@@ -306,6 +306,15 @@ global:
       # controlPlaneHighAvailabilityHelp:
       #   text: Help text
 
+      # #knownConditions - overwrite or add shoot condition descriptions
```
Comment (holgerkoser) [inline charts/gardener-dashboard/values.yaml:309]:
```suggestion
      # # knownConditions - overwrite or add shoot condition descriptions
```

File: charts/gardener-dashboard/charts/runtime/templates/dashboard/configmap.yaml [hunk 5]
```diff
@@ -323,4 +323,8 @@ data:
       {{- if .Values.global.dashboard.frontendConfig.serviceAccountDefaultTokenExpiration }}
       serviceAccountDefaultTokenExpiration: {{ .Values.global.dashboard.frontendConfig.serviceAccountDefaultTokenExpiration }}
       {{- end }}
+      {{- if .Values.global.dashboard.frontendConfig.knownConditions }}
+      knownConditions:
+{{ toYaml .Values.global.dashboard.frontendConfig.knownConditions | trim | indent 8 }}
```
Comment (holgerkoser) [inline charts/gardener-dashboard/charts/runtime/templates/dashboard/configmap.yaml:328]:
```suggestion
      knownConditions: {{ toYaml .Values.global.dashboard.frontendConfig.knownConditions | nindent 8 }}
```

File: charts/__tests__/gardener-dashboard/runtime/dashboard/__snapshots__/deployment.spec.js.snap (hunk 2 above)
Comment (grolu) [reply charts/__tests__/gardener-dashboard/runtime/dashboard/__snapshots__/deployment.spec.js.snap:149]: Yes they fail with current master

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**